app/views/manage/projects/_project_status_form.html.haml

```diff
@@ -0,0 +1,19 @@
+= form_with model: project, url: manage_project_path(project),
+    html: { id: "project-status-form", class: "form", data: { turbo_frame: "project-status-frame" } } do |form|
+  = form.select :status, options_for_select(Project::STATUSES, project.status),
+    { include_blank: false },
+    { id: "project-status-select", class: "form-select", autocomplete: 'off',
+      onchange: 'this.form.requestSubmit()' }
+  - if project.saved_change_to_status?
+    %span.update-flash.visible SAVED
+  - else
+    %span.update-flash.opacity-0 SAVED
+
+:javascript
```

Comment:
The possiblity of reusing this? If so should we pull it out like we did with clipboard?

Reply:
Given the Stimulus learning curve I'm happy to volunteer to do this as a follow up. I'll do it as a single PR afterward so it's a clear example of breaking things out